.code-samples.meilisearch.yaml

```diff
@@ -1209,3 +1209,15 @@ negative_search_2: |-
     -X POST 'http://localhost:7700/indexes/movies/search' \
     -H 'Content-Type: application/json' \
     --data-binary '{ "q": "-\"escape room\"" }'
+analytics_event_send_1: |-
```

Comment:
Ok with it, but so far, no method exists in the integration for this so far..

should we add them for JS and PHP @meilisearch/product-team @brunoocasali?

Reply:
It's your/the product team's call. I think supporting sending analytics events with the SDK could help promoting this feature, but that's just a guess on my part.

I will keep the sample on `.code-samples.meilisearch.yaml` in any case, even if it's `curl`-only at the moment . We don't really lose much from it being there.

Reply:
Yes, I believe it should be contained in the SDKs

Reply:
Thanks for pinging me.

I agree, we should add this feature to the JS and PHP SDKs and include the related code samples in the documentation.

Reply:
I also agree. We just need to find a way to make it very clear to the regular user, that these functions are only available under the cloud subscription + using the edge.

Reply:
Seems we're all aligned on the matter. I will mark this conversation as resolved, since I should be merging the PR soon.

@meilisearch/integration-team, let me know if you need anything else. There's no rush on my side. The docs website will pull the code samples once they're ready, and will continue displaying only the `curl` code sample in the meantime.